base_custom_attribute/models/attribute_set_owner.py

```diff
@@ -2,18 +2,46 @@
 # @author Sébastien BEAU <sebastien.beau@akretion.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
+from lxml import etree
+
 from odoo import api, fields, models
 
 
 class AttributeSetOwnerMixin(models.AbstractModel):
+    """Override the inherited model's fields_view_get() and replace
+    the 'attributes_placeholder' by a group element made of all the Attributes related
+    to the inherited model.
+    Each Attribute will have a conditional invisibility depending on its Attriute Sets.
+    """
+
     _name = "attribute.set.owner.mixin"
     _description = "Attribute set owner mixin"
 
     attribute_set_id = fields.Many2one("attribute.set", "Attribute Set")
 
     @api.model
-    def _build_attribute_view(self):
+    def _insert_attribute(self, eview):
         attributes = self.env["attribute.attribute"].search(
             [("model_id.model", "=", self._name), ("attribute_set_ids", "!=", False)]
         )
-        return attributes._build_attribute_view()
+        return attributes._insert_attribute(eview)
+
+    @api.model
+    def fields_view_get(
+        self, view_id=None, view_type="form", toolbar=False, submenu=False
+    ):
+        result = super(AttributeSetOwnerMixin, self).fields_view_get(
+            view_id=view_id, view_type=view_type, toolbar=toolbar, submenu=submenu,
+        )
+        if view_type == "form":
+            context_include = self._context.get("include_native_attribute")
+            eview = etree.fromstring(result["arch"])
+
+            # Replace <placeholder name='attributes'> by a group of all the model's
+            # Attributes
+            eview_attributes = self.with_context(
+                {"include_native_attribute": context_include}
+            )._insert_attribute(eview)
```

Comment:
pass a string for the "_insert_attribute" method (move the etree.fromstring(result["arch"]) and etree.tostring(eview_attributes, pretty_print=True) into "_insert_attribute"